src/browser/components/ManualLink.test.js

```diff
@@ -39,6 +39,24 @@ const tests = [
   [
     { neo4jVersion: '4.0.0-beta03mr03', chapter: 'driver-manual', page: '' },
     'https://neo4j.com/docs/driver-manual/4.0-preview/'
+  ],
+  [
+    {
+      neo4jVersion: '3.4.11',
+      chapter: 'driver-manual',
+      page: '',
+      minVersion: '4.0.0'
+    },
+    'https://neo4j.com/docs/driver-manual/4.0/'
+  ],
+  [
+    {
+      neo4jVersion: '4.0.0-rc01',
+      chapter: 'driver-manual',
+      page: '',
+      minVersion: '3.5.0'
+    },
+    'https://neo4j.com/docs/driver-manual/4.0-preview/'
   ]
```

Comment:
Could you add one more test, with `minVersion` defined but no `neo4jVersion` defined (simulating a browser that's not connected to a dbms yet).

Reply:
Good catch, thank you! I had to update the component as well to make this work.